go/enclave/main/entry.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+set -e
```

Comment:
This one is confusing me, let's add a couple of comments here on what is happening and why for future references.

Reply:
Good point ! Adding them 👍